.travis.yml

```diff
@@ -75,8 +72,8 @@ matrix:
     addons : *sawaddons   
   allow_failures:
     - os: osx
-    - env: S2N_LIBCRYPTO=openssl-1.1.x-master BUILD_S2N=true TESTS=integration GCC6_REQUIRED=true
     # Optional until we resolve broken remote dependency: https://github.com/awslabs/s2n/pull/615
+    - env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true TESTS=integration GCC6_REQUIRED=true
```

Comment:
I think this can be moved out of allowed_failures, it's passing in the latest build https://travis-ci.org/awslabs/s2n/builds/502316720?utm_source=github_status&utm_medium=notification . Or perhaps there's another reason we think it might be flaky?

Reply:
Let's find out! I'm updated and we'll see how it goes :)